packages/fxa-content-server/app/scripts/views/subscriptions_redirect.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import FormView from './form';
+import Template from 'templates/subscriptions_redirect.mustache';
+
+class SubscriptionsRedirectView extends FormView {
+  mustAuth = true;
+  template = Template;
+
+  initialize (options) {
+    this._currentPage = options.currentPage;
+    this._config = {};
+    if (options && options.config && options.config.subscriptions) {
+      this._config = options.config.subscriptions;
+    }
+  }
+
+  afterRender () {
+    const {
+      managementClientId,
+      managementScopes,
+      managementTokenTTL,
+      managementUrl,
+    } = this._config;
+    return this.getSignedInAccount()
```

Comment:
Let's extract this functionality to a helper that can be used here and from settings. I really don't want to redirect with an OAuth token, the more I think about it, the more I think it's the wrong thing to do. I'd much rather use [an OAuth flow with prompt=none](https://github.com/mozilla/fxa/pull/1150), we'll be secure knowing the user that ends up on the subscription management page is the same as the person who created the token.

Reply:
More on why the extraction. Sorry, once we move to an OAuth flow with prompt=none, I only want to make that update in one place.

Reply:
Yeah, this was literally just copypasta to get to a proof-of-concept

Reply:
Reworked things to extract this to a common module